index.html

```diff
@@ -293,7 +293,7 @@ <h3>
           </p>
 
           <pre class="example nohighlight" title="Creating the DID with optional path and port">
-did:web:example.com%3A3000:user:alice 
+did:web:example.com%3A3000:user:alice
```

Comment:
```suggestion
did:web:example.com%3A3000:user:alice.json
```
This PR is suggesting the above, right? If not, might object. Why aren't we just including file extensions?

Reply:
No, the only reason that line is at all present in this PR, is that my code editor (correctly :) ) drops trailing spaces.

Reply:
I'm not a fan of binding the representation to the subject identifier here either so -1 for me.